cheval.js

```diff
@@ -170,15 +170,18 @@
                                         );
                                     }
                                 } else {
+                                    document.activeElement.blur();
```

Comment:
This change is awesome, thanks for noticing that behavior on iOS and improving it. Would you mind writing a comment (complete sentence if possible) to explain why this is necessary?